src/parser/lexer/mod.rs

```diff
@@ -86,6 +86,8 @@ pub enum Command {
     SetScale,
     ShiftFrequency,
     ShiftPhase,
+    #[strum(to_string = "SWAP-PHASES")]
```

Comment:
Shouldn't be necessary, since the enum is annotated with `serialize_all = "SCREAMING-KEBAB-CASE"`